orm/models.py

```diff
@@ -48,6 +48,11 @@ def __new__(
         return new_model
 
 
+def prepare_order_args(order_args):
+    from sqlalchemy.sql import text
+    return [text(arg) for arg in order_args]
```

Comment:
After trying this out, it seems this isn't sufficient to use `DESC` on more complex datatypes such as datetimes.

May I suggest the more explicit implementation below? I tested this locally and it worked fine e.g. on `.order_by('-created')`.

```python
def prepare_order(model_cls, order):
    prepared = []

    for clause in order:
        if clause.startswith("-"):
            desc = True
            col_name = clause.lstrip("-")
        else:
            desc = False
            col_name = clause

        col = model_cls.__table__.columns[col_name]
        prepared.append(col.desc() if desc else col)

    return prepared
```

Reply:
that war the initial idea, and the reason I've made a helper function in the first place. But my knowledge of sqlalchemy is low. Do you think any other datatypes may cause problems?

Reply:
No problem, I’m also just getting started with the SQLAlchemy core internals. :-)

The documentation highlights .desc() as the recommended way to sorting in descending order: https://docs.sqlalchemy.org/en/13/core/sqlelement.html

So I think this should be good. Maybe @tomchristie has hints on how best this should be tested?